internal/packager/extract.go

```diff
@@ -31,7 +31,7 @@ func findApp() (string, error) {
 	for _, c := range content {
 		if strings.HasSuffix(c.Name(), internal.AppExtension) {
 			if hit != "" {
-				return "", fmt.Errorf("multiple applications found in current directory, specify the application name on the command line")
+				return "", fmt.Errorf("Error: multiple applications found in current directory, specify the application name on the command line")
```

Comment:
There's an e2e test which checks this error. It failed until I added this "Error: "...

Reply:
Oh right, I even realised this in my WIP PR where I wrote in the commit message:
```
    Finally, the idiom in docker/cli (for better or worse) is to include an "Error:
    " prefix in the error string itself, rather than injecting it when printing.
    Since CLI plugins follow the behaviour of the CLI here it is necessary to
    prepend "Error: " to some messages. I've only done exactly those necessary to
    pass the e2e tests, a fuller audit is very likely required.
    
```
(and then forgot about it, obviously)

Feel free to steal some or all of my words for your commit message.

Reply:
I let this fuller audit as an exercise for the reviewers :trollface: